ansible-scylla-node/tasks/main.yml

```diff
@@ -8,6 +8,51 @@
   package_facts:
     manager: auto
 
+- name: Set Scylla package prefix as OSS
+  set_fact:
+    scylla_package_prefix: "scylla"
+  when: scylla_edition == 'oss'
+
+- name: Set Scylla package prefix as Enterprise
+  set_fact:
+    scylla_package_prefix: "scylla-enterprise"
+  when: scylla_edition == 'enterprise'
+
+- name: Check if {{ scylla_package_prefix }} meta-package is installed
+  set_fact:
+    scylla_is_installed: true
+  when: scylla_package_prefix in ansible_facts.packages
+
+- name: Get installed scylla version
+  set_fact:
+    installed_scylla_version: "{{ ansible_facts.packages[scylla_package_prefix][0]['version'] }}"
+  when: scylla_is_installed
+
+- name: Set local scylla_version fact
+  set_fact:
+    scylla_version_to_install: "{{ scylla_version }}"
+
+- name: Get latest Scylla {{ scylla_package_prefix }} version if needed
+  block:
+    - name: Get OSS latest version
+      ansible.builtin.uri:
+        url: "https://repositories.scylladb.com/scylla/check_version"
+        method: GET
+        register: scylla_latest_version
```

Comment:
so there can be mismatch in https://github.com/scylladb/scylla-ansible-roles/blob/master/ansible-scylla-node/requirements.txt
and ansible version 6.7
""Unsupported parameters for (ansible.legacy.uri) module: register."

but more likely tests fail due to extra TAB before register - it should be on the same level as "name"
so please fix that(just move this line few spaces back to "- name" level)